docs/modules/ROOT/pages/hazelcast-platform-operator-external-backup-restore.adoc

```diff
@@ -0,0 +1,500 @@
+= External Backup and Restore with Hazelcast
+:page-layout: tutorial
+:page-product: operator
+:page-categories: Cloud Native
+:page-lang: go, java, node, python
+:page-enterprise: true
+:page-est-time: 20 mins
+:description: Migrate Hazelcast backup data to/from blob storages that cloud providers maintain, within Hazelcast cluster running in Kubernetes.
+
+{description}
+
+== Context
+In this tutorial you will learn:
+
+- Deploying Hazelcast with persistence enabled.
+
+- Creating persistence enabled map in the Hazelcast.
+
+- Backing data up from the map into the external storage.
+
+- Restarting the Hazelcast cluster with the restored data from the external storage.
+
+There are two available options for Persistence feature of Hazelcast Platform Operator:
```

Comment:
I think we can remove all the information below the list and link to our docs page about persistence in the Hazelcast Platform Operator docs.